### PR TITLE
Discord.js V13 Upgrades

### PR DESCRIPTION
**Pull request by WillFJS**
This pull request is for discord.js v13 upgrades.

### CODE REVIEW

**Comment by MidSpike** [comment]
Do not merge without testing

**Comment by Will3126** [comment]
help cmd v13*

**Comment by Will3126** [comment]
Do not review until done. Thanks!

**Review by MidSpike — CHANGES_REQUESTED** [review]
@Will3126 Please `CTRL` + `F` and find any instances of `embed:` and fully replace them with `embeds`

**Review by MidSpike — COMMENTED** [review]
@Will3126 the changes are starting to look good, just remember to keep a sharp eye when making edits

**Comment by Will3126** [comment]
> @Will3126 add trailing commas

will do!

**Comment by Will3126** [comment]
do you see any missing trailing commas

File: src/bot/commands/quick_support.js [hunk 1]
```diff
@@ -205,32 +213,40 @@ module.exports = {
...
+                        iconURL: `${client.user.displayAvatarURL({ dynamic: true })}`,
+                        name: 'Inertia Lighting | Quick Support System',
+                    },
+                    title: `${best_matching_qs_topic.title}`,
+                    description: `${best_matching_qs_topic.support_contents}`,
+                }),
+            ],
+        });
+    },
 }
```
Comment (MidSpike) [inline src/bot/commands/quick_support.js:252]: missing semi-colon

File: src/bot/commands/reload.js [hunk 2]
```diff
@@ -20,15 +20,19 @@ module.exports = {
         const { command_args } = args;
 
         if (command_args.length === 0) {
-            message.channel.send(`You didn't pass any command to reload, ${message.author}!`);
+            message.channel.send({ 
+                content: `You didn't pass any command to reload, ${message.author}!`
```
Comment (MidSpike) [inline src/bot/commands/reload.js:24]: missing comma

File: src/bot/commands/reload.js [hunk 3]
```diff
@@ -37,10 +41,14 @@ module.exports = {
         try {
             const new_command = require(`./${command.name}.js`);
             client.$.commands.set(new_command.name, new_command);
-            message.channel.send(`Command \`${new_command.name}\` was reloaded!`);
+            message.channel.send({ 
+                content: `Command \`${new_command.name}\` was reloaded!`
```
Comment (MidSpike) [inline src/bot/commands/reload.js:45]: missing comma

File: src/bot/commands/reload.js [hunk 4]
```diff
@@ -37,10 +41,14 @@ module.exports = {
...
+            });
         } catch (error) {
             console.error(error);
-            message.channel.send(`There was an error while reloading command \`${command.name}\`:\n\`\`\`${error.message}\`\`\``);
+            message.channel.send({ 
+                content: `There was an error while reloading command \`${command.name}\`:\n\`\`\`${error.message}\`\`\``
```
Comment (MidSpike) [inline src/bot/commands/reload.js:50]: missing comma

File: src/bot/discord_client.js [hunk 5]
```diff
@@ -11,7 +11,20 @@ const Discord = require('discord.js');
...
 const client = new Discord.Client({
-    disableMentions: 'everyone',
+    allowedMentions: {
+        parse: [
+            'users',
+            'roles',
+        ],
+        repliedUser: true,
+    },
+    intens: [
```
Comment (MidSpike) [inline src/bot/discord_client.js:21]: intents is misspelled

**Review by MidSpike — DISMISSED** [review]
please fix the things I commented on

File: src/bot/handlers/welcome_message_handler.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+            content: message_contents,
+        }).catch(console.warn);
+    }
+}
+
+//---------------------------------------------------------------------------------------------------------------//
+
+module.exports = {
+    welcomeMessageHandler,
+};
```
Comment (MidSpike) [inline src/bot/handlers/welcome_message_handler.js:62]: missing newline at end of file

**Review by MidSpike — CHANGES_REQUESTED** [review]

**Review by MidSpike — COMMENTED** [review]
after the fixes I made, things are looking good

**Comment by MidSpike** [comment]
@WillFJS have you tested this branch yet?

**Comment by MidSpike** [comment]
> @WillFJS have you tested this branch yet?

@WillFJS re-pinging you for this

**Comment by MidSpike** [comment]
@WillFJS
ca7ef4f2-c6ad-4449-b4e2-97591e78f236-min

**Review by MidSpike — APPROVED** [review]
After initial testing, and fixing, everything looks ready to be used.

**Comment by MidSpike** [comment]
@rossloika please review this

**Review by rossloika — APPROVED** [review]

**Comment by MidSpike** [comment]
@itsdrawn please review

**Comment by MidSpike** [comment]
@rossloika @WillFJS we need to test this soon

**Review by MidSpike — APPROVED** [review]

**Review by DJHARDY2K21 — APPROVED** [review]
Great work!

**Comment by DJHARDY2K21** [comment]
Make sure you have changed the names for the events as there is a minor changes with the events. *Old ones are becoming deprecated*

**Review by itsdrawn — APPROVED** [review]
everything looks good

**Comment by MidSpike** [comment]
@WillFJS you must test this soon

**Comment by WillFJS** [comment]
looks good

**Comment by MidSpike** [comment]
> and fix the merge conflicts

@WillFJS You need to fix the merge conflicts

**Comment by MidSpike** [comment]
@Inertia-Lighting/discord-bot-api-server 

The time has come for finalizing our reviews on this pull request as Discord.js just released v13

**Review by MidSpike — APPROVED** [review]

**Review by itsdrawn — APPROVED** [review]
lgtm

**Review by rossloika — APPROVED** [review]
After some time of reading it looks good to me.

**Pull request merged**